sysutils/borgbackup/Portfile

```diff
@@ -35,6 +35,7 @@ depends_build-append    port:py${python.version}-setuptools_scm \
 depends_lib-append      path:lib/libssl.dylib:openssl \
                         port:lz4 \
                         port:py${python.version}-msgpack \
```

Comment:
this would be a good time to re-evaluate the dependencies. 

- `py-msgpack`: currently it will use the bundled version; this is a good idea here I think as in the past the port often broke upon updates of `py-msgpack`. I would recommend to remove the dependency here. If we really want it to use the system version, one has to set `prefer_system_msgpack = True` in `src/borg/helpers.py`.

- the default is to use system dependencies of `lz4` (already listed), `libb2`, `xxhash` and `zstd`. These could/should be added as  dependencies here. 

- `py-setuptools` is no longer needed at runtime; the first choice is to use `importlib.metadata` which is part of the default library in Python 3.9.

so I would suggest to change `depends_lib-append` to:

```
depends_lib-append    path:lib/libssl.dylib:openssl \
                      port:lz4 \
                      port:libb2 \
                      port:py${python.version}-packaging \
                      port:xxhash \
                      port:zstd

Reply:
I have almost zero knowledge about the macports codebase, if you or anyone could take over, I think it would be much easier than guiding me through it. I don't care about attribution, just didn't want it to be lost.

Reply:
@lambdaupb we always appreciate contributions so there is no problem if you need some guidance!

Your suggested change is perfectly fine and will fix the error you're seeing. The only thing that *needs* to be changed for this PR to be merged is to change the value of the `revision` field to `1`. Since the version didn't change (that's typically when the port will be listed as `outdated` and user will update it) increasing the revision will have the same effect. So if you just want to make that change and push here, we could merge the PR. 

When I looked at the dependencies listed by upstream I noticed that it will use system-versions of quite a few of them. That means that if the user has them installed they will be used; if not, it will use the bundled versions. That behavior violates the principle of [reproducible builds](https://trac.macports.org/wiki/ReproducibleBuilds) and should be fixed. You can do that as part of this PR or I can take of it later. That's up to you, just let us know!